lua/keymappings.lua

```diff
@@ -9,94 +9,94 @@ local opts = {
 local keymaps = {
   insert_mode = {
     -- I hate escape
-    { "jk", "<ESC>" },
-    { "kj", "<ESC>" },
-    { "jj", "<ESC>" },
+    ["jk"] = { "<ESC>" },
+    ["kj"] = { "<ESC>" },
+    ["jj"] = { "<ESC>" },
     -- Move current line / block with Alt-j/k ala vscode.
-    { "<A-j>", "<Esc>:m .+1<CR>==gi" },
-    { "<A-k>", "<Esc>:m .-2<CR>==gi" },
+    ["<A-j>"] = { "<Esc>:m .+1<CR>==gi" },
+    ["<A-k>"] = { "<Esc>:m .-2<CR>==gi" },
     -- navigation
-    { "<A-Up>", "<C-\\><C-N><C-w>k" },
-    { "<A-Down>", "<C-\\><C-N><C-w>j" },
-    { "<A-Left>", "<C-\\><C-N><C-w>h" },
-    { "<A-Right>", "<C-\\><C-N><C-w>l" },
+    ["<A-Up>"] = { "<C-\\><C-N><C-w>k" },
+    ["<A-Down>"] = { "<C-\\><C-N><C-w>j" },
+    ["<A-Left>"] = { "<C-\\><C-N><C-w>h" },
+    ["<A-Right>"] = { "<C-\\><C-N><C-w>l" },
   },
 
   normal_mode = {
     -- Better window movement
-    { "<C-h>", "<C-w>h" },
-    { "<C-j>", "<C-w>j" },
-    { "<C-k>", "<C-w>k" },
-    { "<C-l>", "<C-w>l" },
+    ["<C-h>"] = { "<C-w>h" },
+    ["<C-j>"] = { "<C-w>j" },
+    ["<C-k>"] = { "<C-w>k" },
+    ["<C-l>"] = { "<C-w>l" },
 
     -- Resize with arrows
-    { "<C-Up>", ":resize -2<CR>" },
-    { "<C-Down>", ":resize +2<CR>" },
-    { "<C-Left>", ":vertical resize -2<CR>" },
-    { "<C-Right>", ":vertical resize +2<CR>" },
+    ["<C-Up>"] = { ":resize -2<CR>" },
+    ["<C-Down>"] = { ":resize +2<CR>" },
+    ["<C-Left>"] = { ":vertical resize -2<CR>" },
+    ["<C-Right>"] = { ":vertical resize +2<CR>" },
 
     -- Tab switch buffer
     -- { "<TAB>", ":bnext<CR>" },
```

Comment:
Looks like there are some leftover comments here as well.  Do you want to remove them or update them to the new syntax?   I'm for removing them.

Reply:
Too late, but to answer you I did it manually

Reply:
This is the perfect situation for macros.  

![Uploading macro.gif…]()

Reply:
@Tasty, you're oldschool!!